modules/auxiliary/admin/http/ulterius_file_download.rb

```diff
@@ -0,0 +1,94 @@
+require 'zlib'
```

Comment:
Not sure you need this. You can check by running `pry` within the module context and seeing if `Zlib` exists.